mixins/trim-whitespace/trim-whitespace-mixin.js

```diff
@@ -0,0 +1,32 @@
+import { TrimWhitespaceCore } from './trim-whitespace-core.js';
+
+export const TrimWhitespaceMixin = superclass => class extends superclass {
+	static get properties() {
+		return {
+			trimWhitespaceDeep: { type: Boolean, attribute: 'trim-whitespace-deep', reflect: true },
```

Comment:
What's the motivation for this being a reactive property that would be exposed to consumers of a component using this mixin vs. just being a regular property that the component itself would decide when to use?

Just thinking in that scenario, the component would set it in its constructor and then you'd defer creating `TrimWhitespaceCore` until `connectedCallback`. There'd be no ability to change the value later obviously.

Was there a scenario where the component itself doesn't know if it wants deep whitespace trimming and it would need to be left up to the consumer?

Reply:
No particularly good reason 😄. I asked myself the same question when I started writing the readme. I'd be happy to remove the reactive property - would you think:

1. Still keep it as a property (`.trimWhitespaceDeep`) on the object, and still set it in the constructor as the readme shows?
2. Or change it to a `static get trimWhitespaceDeep()` which can then be set on the inheriting object?

The advantage of number 2 is that it would be less likely someone would try to change the property after initializing, I think.